src/app/routes/applications/governance-list/governance-list.component.html

```diff
@@ -172,6 +172,12 @@
                 <span>Create Role</span>
               </button>
 
+              <!-- Create Staking Pool -->
+              <button mat-menu-item class="btn-color-primary" (click)="openStakingPool(element)">
```

Comment:
Please add `*appIsFeatureEnabled` here. It works like ngIf, but listens on feature flag. So it will be visible on switchboard-dev but not on volta (prod environment). I think you will need to rebase/merge changes from develop branch.